src/containers/Home.jsx

```diff
@@ -57,6 +58,9 @@ class Home extends Component {
               })}
             />
           </Box>
+          if (wallet.length === 0) {
+            <NoData />
+          }
           <List selectable>
```

Comment:
I am thinking of moving this into its own component so that it can be included in the conditional above